server/protocols/jmap/src/main/java/org/apache/james/jmap/JMAPServlet.java

```diff
@@ -90,6 +90,15 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws S
         }
     }
 
+    private void sendResponses(HttpServletResponse response, List<Object[]> responses) {
+        try {
+            objectMapper.writeValue(response.getOutputStream(), responses);
+        } catch (IOException e) {
+            LOGGER.info("Error sending response", e);
+            response.setStatus(SC_BAD_REQUEST);
```

Comment:
So if we don't succeed to write the response back, it is a 400?

Reply:
My goal is to keep the same behavior, just to lower the logging level.

Reply:
for sure, but changing the level means we are ok with the error handling in the first place, don't you think ?

Reply:
Well, a failure is a failure, it's just a way to inform the admin that (s)he has nothing to do/worry about.

Reply:
We agree, so here we have several problems:
* if we have an IOException because of json transformation, we want the admin to know and we don't want the user to receive a BAD_REQUEST but an INTERNAL_ERROR
* if we have an IOException because the outputstream is broken, we don't care

Reply:
- if we have an IOException for any other reason, that is an error log + a 500 error

Reply:
I'm lost, what should I do?

Reply:
you should check the root cause of the IOException and answer the client accordingly to what is said above